### PR TITLE
#1984 - restyle share post modal

### LINKED ISSUE #1984: Icons in the share post modal looks too close to each other
# The icons on the share post modal looks to close to each other
- My suggestions are that if the width of the Modal is the same as that of Create Post Modal, then it will look better
- My second thought is the text input where the URL is displayed is looks like a plain HTML input, it would look better if we make it on liner input and copy-link button next to it.

Thank you!

Screen Shot 2020-11-29 at 8 52 09 PM

### PR DESCRIPTION
**Pull request by aleksarad**
### What does this pull request aim to achieve? 💯
Restyles share post modal according to design team mockup
- Replaces react-share icons with custom icons
- Adds copy to clipboard function to share post link
- Updates the en_US file to include updated text and new text

_Please be concise and link any related issue(s):_
resolves 1984

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/components/Button/PostShareButton.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const PostShareButton = styled(BaseButton)`
+  padding: 1rem 2rem;
+  margin-left: 0.5rem;
+  border-radius: 4.6rem;
+  border: 1px solid ${white} !important;
+  font-family: ${display};
+  font-size: 1.3rem;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
```
Comment (CamilaArias) [inline client/src/components/Button/PostShareButton.js:16]: don't need lines 14 to 16

File: client/src/components/Button/PostShareButton.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import BaseButton from "./BaseButton";
+import { theme } from "constants/theme";
+const { royalBlue, white } = theme.colors;
+const { display } = theme.typography.font.family;
+
+const PostShareButton = styled(BaseButton)`
+  padding: 1rem 2rem;
+  margin-left: 0.5rem;
+  border-radius: 4.6rem;
+  border: 1px solid ${white} !important;
```
Comment (CamilaArias) [inline client/src/components/Button/PostShareButton.js]: use royalBlue because white makes it look a bit odd :)

**Review by CamilaArias — CHANGES_REQUESTED** [review]
It's looking good @aleksarad!
On the visual side just one comment, also reddit and whatsapp icons aren't showing the svg it seems so they look off.

The email button doesn't work, it also doesn't work in prod, is there a way to fix it or should we just remove it?

**Comment by aleksarad** [comment]
thanks @CamilaArias :) Will implement those changes.

The email button works for me on this branch and in production - it just pops open my email app with the link in the body.

For the icons - those are svgs directly from the figma, the two that look funky/fuzzy are the only ones using data URIs/images inside the svg (instead of vector) so maybe that's what's causing a difference in how they're displaying but I'm really not sure. I'll ask for help in slack regarding the svgs.

**Comment by CamilaArias** [comment]
Oh gotcha @aleksarad it doesn't work for me then 🤣 I'll look at those icons on Figma.

Edit: Fixed [icons on Figma](https://www.figma.com/file/EMKcg0eTnOlr7bRZGFdnZu/Web-app-product-spec?node-id=515%3A12) :)

**Comment by aleksarad** [comment]
Latest push updates the icons, reviewers - let me know if there's any other changes that need to be made :)

**Review by Zaydme — APPROVED** [review]
LGTM! great job @aleksarad!

**Review by rabbi08 — APPROVED** [review]
Looks good. Thanks

**Pull request merged**